### PR TITLE
RHDEVDOCS-4282 - Doc: Pipelines Usability & Customer Happiness

### PR DESCRIPTION
**Pull request by smunje1**
[RHDEVDOCS-4282](https://issues.redhat.com//browse/RHDEVDOCS-4282): Doc: Pipelines Usability & Customer Happiness

Aligned team: Dev Tools
OCP version for cherry-picking: enterprise-4.12
JIRA issues: [RHDEVDOCS-4282](https://issues.redhat.com//browse/RHDEVDOCS-4282)
Preview pages: [Download to see the preview in your browser](https://drive.google.com/file/d/1314yDZj6H6zMJdW7g-L2TMW1MwWBGBho/view?usp=share_link)
SME Review: @vikram-raj 
QE review: @sanketpathak 
Peer-review: @sounix000 and @DebarghoGhosh

### CODE REVIEW

**Comment by smunje1** [comment]
Hello @vikram-raj 
Could you help with the SME review for this PR? 

Thank you!

**Review by vikram-raj — COMMENTED** [review]
LGTM

**Comment by smunje1** [comment]
Hello @sanketpathak 
Could you help with the QE review for this PR?

Thank you!

File: modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc [hunk 1]
```diff
@@ -20,10 +20,10 @@ The *Pipelines* view in the *Developer* perspective lists all the pipelines in a
...
 . Click the required pipeline to see the *Pipeline details* page.
 +
-By default, the *Details* tab displays a visual representation of all the all the serial tasks, parallel tasks, `finally` tasks, and when expressions in the pipeline. The tasks and the `finally` tasks are listed in the lower right portion of the page. Click the listed *Tasks* and *Finally tasks* to view the task details.
+By default, the *Details* tab displays a visual representation of all the all the serial tasks, parallel tasks, `finally` tasks, and when expressions in the pipeline. The tasks and the `finally` tasks are listed in the lower right portion of the page. Click the listed *Tasks* and *Finally tasks* to view the task details. You can use the zoom in, zoom out, fit to screen, and reset view features using the standard icons, which are in the lower left corner of the *Pipeline details* page visualization. In addition, you can change the zoom factor of the pipelines visualization using the mouse wheel. Also, you can hover over the tasks and see the task details.
```
[the remark this answers is not in the document]
Comment (smunje1) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:23]: This suggestion is for the existing documentation. I believe, we only need to remove the twice repetition as mentioned in the first bullet point. Also, we should avoid using the **bottom** word usage it seems (https://www.ibm.com/docs/en/ibm-style?topic=word-usage)
image

"By default, the *Details* tab displays a visual representation of all the serial tasks, parallel tasks, `finally` tasks, and when expressions are in the pipeline. The tasks and the `finally` tasks are listed in the lower right portion of the page. Click the listed *Tasks* and *Finally tasks* to view the task details. You can use the zoom in, zoom out, fit to screen, and reset view features using the standard icons, which are in the lower left corner of the *Pipeline details* page visualization. In addition, you can change the zoom factor of the pipeline visualization using the mouse wheel. Also, you can hover over the tasks and see the task details."

@sounix000 @DebarghoGhosh  Could you please let me know your thoughts? Thank you!

[the remark this answers is not in the document]
Comment (sounix000) [reply modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc:23]: SUGGESTION

By default, the **Details** tab displays a visual representation of all the `serial` tasks, `parallel` tasks, `finally` tasks, and `when` expressions in the pipeline. The tasks and the `finally` tasks are listed in the lower right portion of the page. 

To view the task details, click the listed **Tasks** and **Finally** tasks. In addition, you can do the following:

- Use the zoom in, zoom out, fit to screen, and reset view features using the standard icons displayed in the lower left corner of the **Pipeline details** visualization page. 
- Change the zoom factor of the pipeline visualization using the mouse wheel. 
- Hover over the tasks and see the task details.

**Comment by smunje1** [comment]
@sounix000 @DebarghoGhosh Could you send your approvals if there are no more comments?

I have incorporated the suggestion. Thanks!

**Comment by smunje1** [comment]
After the updates, screenshot:
image

**Comment by smunje1** [comment]
After the updates, screenshot:
image

Cc @vikram-raj @sanketpathak

**Pull request merged**